src/Main.java

```diff
@@ -1,7 +1,63 @@
+import java.util.Arrays;
+
 //TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
-
+        System.out.println("Task01");
+        int[] array1=new int[] {1,2,3};
```

Comment:
этот способ равносилен такому
```suggestion
        int[] array1 = {1,2,3};
```
а требовалось создать массив длины 3 и затем заполнить его числами (была даже подсказка)